### ISSUE TITLE
Add Configurable Non-Anonymous/Single Hop Mode

### ISSUE THREAD

**Issue by rfilmyer** [cmehay/docker-tor-hidden-service#38]
I have been using this container to run a number of onion services, but for my use case I am more interested in performance than _server-side_ anonymity. So I'm interested in [Tor's Single Hop + Non-Anonymous Mode](https://blog.torproject.org/whats-new-tor-0298), which allows onion services to make this tradeoff (reducing the client-server connection from 6 hops to 4), within certain limitations. It still preserves client anonymity and is still more secure than accessing a site on the open web (thanks to maintaining end-to-end encryption).

This feature is enabled in a `.torrc` file with the following two settings:
```
HiddenServiceNonAnonymousMode 1
HiddenServiceSingleHopMode 1
```

On first thought, this could be implemented with an environment variable and then a corresponding spot in the `.torrc` jinja template.

Thanks for putting together the container!

**Comment by cmehay** [comment]
There is many feature requests to add some options in `torrc` jinja template. I think I will add a simple generic option passthrough to add any needed option more easily.

**Comment by cmehay** [comment]
Fixed by #39
